docs/fusion.md

```diff
@@ -54,75 +142,30 @@ The AWS S3 bucket should be configured with the following IAM permissions:
 }
 ```
 
-## Use cases
+### Google Cloud
 
-### Local execution with S3 bucket as work directory
+Fusion provides built-in support for [Google Storage](https://cloud.google.com/storage?hl=en)
+when running in Google Cloud.
 
-Fusion file system allows the use of an S3 bucket as a pipeline work directory with the Nextflow local executor. This configuration requires the use of Docker (or similar container engine) for the execution of your pipeline tasks.
-
-The AWS S3 bucket credentials should be made accessible via standard `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
-
-The following configuration should be added in your Nextflow configuration file:
-
-```groovy
-docker {
-    enabled = true
-}
-
-fusion {
-    enabled = true
-    exportStorageCredentials = true
-}
+The support for Google does not require any specific setting other then enabling Wave and Fusion in your Nextflow
+configuration. For example:
 
-wave {
-    enabled = true
-}
 ```
-
-Then you can run your pipeline using the following command:
-
-```bash
-nextflow run <YOUR PIPELINE> -work-dir s3://<YOUR BUCKET>/scratch
+process.executor = 'google-batch'
+wave.enabled = true
+fusion.enabled = true
+tower.accessToken = '<your platform access token>'
 ```
 
-Replace `<YOUR PIPELINE>` and `<YOUR BUCKET>` with a pipeline script and bucket or your choice, for example:
+Then run your pipeline using the usual command:
 
-```bash
-nextflow run https://github.com/nextflow-io/rnaseq-nf -work-dir s3://nextflow-ci/scratch
 ```
-
-### AWS Batch execution with S3 bucket as work directory
-
-Fusion file system allows the use of an S3 bucket as a pipeline work directory with the AWS Batch executor. The use of Fusion makes obsolete the need to create and configure a custom AMI that includes the `aws` command line tool, when setting up the AWS Batch compute environment.
-
-The configuration for this deployment scenario looks like the following:
-
-```groovy
-fusion {
-    enabled = true
-}
-
-wave {
-    enabled = true
-}
-
-process {
-    executor = 'awsbatch'
-    queue = '<YOUR BATCH QUEUE>'
-}
-
-aws {
-    region = '<YOUR AWS REGION>'
-}
+nextflow run <your pipeline> -work-dir gs://<your google bucket>/scratch
 ```
 
-Then you can run your pipeline using the following command:
+[TODO add some notes on Google Storage permission and the need for SSD disk and instance type(?)]
```

Comment:
```suggestion
When using Fusion, if the `process.disk` is not set, Nextflow will attach a single local SSD disk to the machine. The size of this disk can be much lower than the actual needs of your pipeline processes because Fusion uses it only as a temporal cache. Fusion is also compatible with other types of `process.disk`, but better performance is achieved when using local SSD disks.
```